massa-graph/src/export_active_block.rs

```diff
@@ -13,7 +13,7 @@ use serde::{Deserialize, Serialize};
 /// Fields that can be easily recomputed were left out
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct ExportActiveBlock {
-    /// The block itself, as it was created
+    /// The id of the block.
```

Comment:
The comment has been changed but it still hold the whole block.

Reply:
Changed here : https://github.com/massalabs/massa/pull/2233#discussion_r821700545